### ISSUE TITLE
Sound issues with PSP mini EDGE (NPUZ00251)

### ISSUE THREAD

**Issue by Peduls** [hrydgard/ppsspp#10776]
What happens?
The in-game music is distorted and slow. I've also tested this with the Europe release.

What *should* happen?
Music sounds clear and normal on actual PSP.

What hardware and operating system are you running PPSSPP on? GPU might matter if it's a graphical issue.
Operating System
	Windows 10 Enterprise 2016 LTSB 64-bit
CPU
	Intel Core i7 4790K @ 4.00GHz
	Haswell 22nm Technology
RAM
	16.0GB Dual-Channel DDR3 @ 799MHz (11-11-11-28)
Motherboard
	ASUSTeK COMPUTER INC. Z97-A (SOCKET 1150)
Graphics
	Acer S231HL (1920x1080@60Hz)
	4095MB NVIDIA GeForce GTX 1070 (EVGA)
Storage
	465GB Samsung SSD 850 EVO 500GB (SSD)
	2794GB TOSHIBA HDWD130 (SATA)
Optical Drives
	ASUS DRW-24B1ST j
Audio
	High Definition Audio Device

**Comment by LunaMoo** [comment]
Probably a duplicate(another case to be exact) of #9379 if you want, you can try my hackish branch build from [here](https://github.com/LunaMoo/ppsspp/releases) it removes the sound speed hack which logic causes the slowdown and also clamps sound which forces volume to hearable levels, helps the other two mini games with that problem.

**Comment by hrydgard** [comment]
I thought we had solved the volume issue at least?

**Comment by LunaMoo** [comment]
Oh yeah I just wasn't testing that, volume might be fine, but the sound plays X times slower, I tested and in this case it's I believe x4 times slower, our "sound speed hack" helps with x2 times slower cases only so it only partially helps, but that whole concept is incorrect.  My change from [here](https://github.com/hrydgard/ppsspp/issues/9379#issuecomment-307278409) removing the "speed hack" and leaving AuCtx::AuDecode() check simpler works fine for it without(as far as I can tell from using it from a long time) affecting any other games negatively.

 Either way I just confirmed it's another case of the mentioned issue, so I'll close this.
